core/values/consensus.go

```diff
@@ -0,0 +1,56 @@
+package values
```

Comment:
1. Please avoid putting new code into the `./core` package or its subpackages, which came from Ethereum and we apply a licensing (LGPL) exception there.
2. I don't think congregating constants into a package simply because they are constants is not a good idea.  Definitions should stay with the actual code to which they belong or at least which uses them the most.  I think the old location (`staking` and `consensus` packages) are more suitable.  (Plus, `types.StakingDirective` reads less natural than `staking.Directive`, for example.)

Could you consider moving them back to where they were?